### ISSUE TITLE
kivy.utils.Platform needs to support detecting *BSD systems

### ISSUE THREAD

**Issue by tony** [kivy/kivy#3560]
```
$ python
Python 2.7.10 (default, Jul  5 2015, 01:15:55) 
[GCC 4.2.1 Compatible FreeBSD Clang 3.4.1 (tags/RELEASE_34/dot1-final 208032)] on freebsd10
Type "help", "copyright", "credits" or "license" for more information.
>>> import kivy.utils
[INFO              ] [Logger      ] Record log in /home/tony/.kivy/logs/kivy_15-08-05_8.txt
[INFO              ] [Kivy        ] v1.9.1-dev
[INFO              ] [Python      ] v2.7.10 (default, Jul  5 2015, 01:15:55) 
[GCC 4.2.1 Compatible FreeBSD Clang 3.4.1 (tags/RELEASE_34/dot1-final 208032)]
>>> kivy.utils.platform
platform name: 'unknown' from: 
<kivy.utils.Platform object at 0x804771690>
```

A lot of the behavior may fall under the linux canopy, and macosx if kqueue for file watching ever comes into play (haven't seen internals yet).

Related is the #3558 issue.

**Comment by tony** [comment]
To follow up with #3559, https://github.com/kivy/kivy/pull/3559#issuecomment-128249571.

``python -c "import sys; print sys.platform"`` -> ``freebsd10``
``python3 -c "import sys; print(sys.platform)"`` -> ``freebsd10``

**Comment by akshayaurora** [comment]
Fixed by 96136ef7480ca5e56003870e8a43d1cfab8ef6bf
Currently we do not differentiate between linux/freebsd, but if we do need to do something specific for BSD I guess we would then look into detecting as free/open/...bsd.
